spec/factories/cohorts.rb

```diff
@@ -3,5 +3,9 @@
 FactoryBot.define do
   factory :cohort do
     start_year { Faker::Number.unique.between(from: 2021, to: 2100) }
+
+    trait :current do
+      start_year { 2021 }
+    end
```

Comment:
This is to match definition of `Cohort.current` which is also hard-coded to 2021 at the moment.